admin/jenkins.py

```diff
@@ -87,9 +87,12 @@
         _run_cmd("python/bin/pyomo install-extras", shell=True)
     elif _run_cmd is subprocess.check_output:
         output = _run_cmd("python/bin/pyomo install-extras", shell=True)
-        if hasattr(output, 'encode'):
-            output = output.encode('utf-8','replace')
-        print(output.decode('utf-8'))
+        try:
+            print(output)
```

Comment:
Why does it fix the unicode behavior to try to print the output before the `encode` and `decode`?

Reply:
The encode / decode part is needed for Python3 support.  I could never get it to cleanly back-port to 2.7.  This change (adding the raw `print()`) is just throwing up my hands.  This is a test driver and not core code, so it is more important that it just works than it being clean/efficient.